src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -116,6 +118,8 @@ export class MeetingManager implements AudioVideoObserver {
 
   logger: Logger | undefined;
 
+  meetingEventObservers: ((name: EventName, attributes: EventAttributes) => void)[] = [];
```

Comment:
How about using a `Set` so that you can remove an observer in O(1)?

Reply:
Yup makes sense, updated to use `Set`.